src/app/services/distance.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({ providedIn: 'root' })
+export class DistanceService {
+
+  convertMetersToKilometers(meters: number): string {
```

Comment:
nice  idea, but it is not recommended in object oriented paradigm. How about you make it just a function? or even better, learn about custom pure pipes: https://angular.io/guide/pipes

Reply:
I would recommend to keep services for specific actions like data fetching, later we will talk about application layer and it will have another use case. But case like this is not meant for service - too simple and it is better to keep your constructor dependencies low because it leads to tight coupling